sw/device/silicon_creator/manuf/lib/flash_info_fields.c

```diff
@@ -83,6 +83,27 @@ const flash_info_field_t kFlashInfoFieldCdi1AttestationKeySeed = {
     .byte_offset = (2 * kAttestationSeedBytes),
 };
 
+const flash_info_field_t kFlashInfoFieldTpmEkAttestationKeySeed = {
+    .partition = 0,
+    .bank = 0,
+    .page = 4,
+    .byte_offset = (3 * kAttestationSeedBytes),
+};
+
+const flash_info_field_t kFlashInfoFieldTpmCekAttestationKeySeed = {
+    .partition = 0,
+    .bank = 0,
+    .page = 4,
+    .byte_offset = (4 * kAttestationSeedBytes),
+};
+
+const flash_info_field_t kFlashInfoFieldTpmCikAttestationKeySeed = {
+    .partition = 0,
+    .bank = 0,
+    .page = 4,
+    .byte_offset = (5 * kAttestationSeedBytes),
```

Comment:
I think this scheme would be more robust to define a structure describing the info page layout, verifying that the structure fits into the page, etc.

Could be a future improvement, this patch naturally follows the existing approach.

Reply:
Good idea, I captured in a separate issue: #22768.